wsclient/src/main/java/org/hpccsystems/ws/client/platform/WorkunitInfo.java

```diff
@@ -382,26 +523,49 @@ public WorkunitInfo setOriginalEclWatchUrl(String serverhost) {
 	    return applicationValues;
 	}
 	
-	public ECLException[] getRawExceptions()
+	public org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ECLException[] getRawExceptions()
 	{
 	    if (this.exceptions.size()==0) {
 	        return null;
 	    }
-	    ECLException[] raw=new ECLException[this.exceptions.size()];
+	    org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ECLException[] raw=
+	            new org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ECLException[this.exceptions.size()];
 	    for (int i=0; i < exceptions.size();i++)
 	    {
 	        raw[i]=exceptions.get(i).getRaw();
 	    }
 	    return raw;
 	}
 	
-	public ApplicationValue[] getRawApplicationValues() 
+	public org.hpccsystems.ws.client.gen.wsworkunits.v1_73.ArrayOfEspException getRawArrayOfEspExceptions()
```

Comment:
If we have that new hpccexceptions class, shouldn't we use it here?

Reply:
yes, but I think that should be a separate PR after this one